### PR TITLE
Update issue templates

### PR DESCRIPTION
**Pull request by gee2450**
issue templates 수정

### CODE REVIEW

**Comment by r4pidstart** [comment]
yes.

**Review by r4pidstart — COMMENTED** [review]
yes

**Comment by BaeJinho4028** [comment]
good

**Review by BaeJinho4028 — APPROVED** [review]
good

**Comment by r4pidstart** [comment]
yes

**Review by r4pidstart — CHANGES_REQUESTED** [review]
yes

File: .github/ISSUE_TEMPLATE/기능-관련-이슈-생성-템플릿.md [hunk 1]
```diff
@@ -0,0 +1,18 @@
+---
+name: 기능 관련 이슈 생성 템플릿
+about: '해당 이슈 생성 템플릿을 사용해서 이슈를 생성하세요. '
+title: "[FEATURE]"
```
Comment (r4pidstart) [inline .github/ISSUE_TEMPLATE/기능-관련-이슈-생성-템플릿.md:4]: no

**Review by r4pidstart — APPROVED** [review]

**Pull request merged**